ownership-chain/node/src/chain_spec.rs

```diff
@@ -181,7 +181,7 @@ fn testnet_genesis(
 		},
 		collator_selection: laos_ownership_runtime::CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			candidacy_bond: laos_ownership_runtime::ExistentialDeposit::get() * 16 + 1,
```

Comment:
the first part of multiplication resolves to 0 anyway
```suggestion
			candidacy_bond: laos_ownership_runtime::WEI,
```

Reply:
### Issues with Using `laos_ownership_runtime::WEI` for `candidacy_bond`

1. **Contextual Inappropriateness**: 
   - WEI, as a denomination, is specific to Ethereum. Applying it within a Substrate-based system could lead to inconsistencies or errors, given the distinct economic models between the two systems.

2. **Potential Inconsistency with `ExistentialDeposit`**:
   - Setting the `candidacy_bond` to a constant value like WEI could create a situation where it is less than the `ExistentialDeposit`. Such a discrepancy could lead to undesired behaviors.

Reply:
> 1. Contextual Inappropriateness:
I agree that WEI is not completely appropriate, but I suggested it for the sake of having some relation to constants defined in the runtime. Ideally, there should be a visible `pub const COLLATOR_CANDIDACY_BOND: u128` in the runtime or the value should be at least tied to a constant in the runtime. In any case, using WEI is still better than having simply 1, imo.
> 2. Potential Inconsistency with ExistentialDeposit:
the point of the PR is to make `ED` zero, why would there be a situation where `candidacy_bond` is less than zero?

Reply:
>  it's crucial to maintain the dependency of candidacy_bond on ExistentialDeposit. Even if ExistentialDeposit is set to zero in current configuration, using a fixed constant for candidacy_bond introduces a risk.

why is it `crucial` to maintain dependency and why there should be a dependency? ED was simply used because there was a need to have some unambiguous value. There is no special relationship between `candidacy_bond` and `ED`. In the same way, you can use `UNIT` or any other constant from runtime to derive the value of `candidacy_bond`.

in any case, I looked at other parachains, some set it to a big value, like astar:

https://github.com/AstarNetwork/Astar/blob/35f6433780025c7e4012afecb6cb3682ca249bd7/bin/collator/src/parachain/chain_spec/astar.rs#L137

some others set it to 0:

https://github.com/AcalaNetwork/Acala/blob/cab5cd8a8f234a0931bec4a854e805fdb896ffc6/node/service/src/chain_spec/acala.rs#L209

So, I suggest we either have it explicitly as `Zero::zero()` or have some non-zero value defined clearly